splight_lib/settings.py

```diff
@@ -58,12 +62,13 @@ class SplightSettings(BaseSettings, Singleton):
     SPLIGHT_PLATFORM_API_HOST: str = "https://api.splight-ai.com"
 
     # Parameters for local environment
+    # TODO: to deprecate this and its effects
     LOCAL_ENVIRONMENT: bool = False
     CURRENT_DIR: Optional[str] = None
 
-    # Parameters for Buffered Datalake Client
+    # Parameters for the datalake client
     # Review if is better to use another class for only the DL Client settings
-    USE_BUFFER: bool = True
+    DL_CLIENT_TYPE: str = DL_DEFAULT_CLIENT_TYPE
```

Comment:
Probably this should be an Enum and we should use that enum everywhere
```python
class DatalakeClientType(str, Enum):
    LOCAL = "local"
    SYNC = "sync"
    BUFFERED_SYNC = "buffered_sync"
    BUFFERED_ASYNC = "buffered_async"
```

so here we use
```python
class SplightSettings(BaseSettings, Singleton):
    ...
    DL_CLIENT_TYPE: DatalakeClientType = DatalakeClientType.BUFFERED_ASYNC
 ```